microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -121,6 +123,8 @@ When you are done, learn how to work with the simulator through [Apple Developer
 <!-- ------------------------------ -->
 #### Microsoft Edge (EdgeHTML)
```

Comment:
delete section "Microsoft Edge (EdgeHTML)" & link to MDN instead? 
@zoherghadyali @bwalderman 

rendered article section: 
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers?branch=pr-en-us-3210#microsoft-edge-edgehtml

Reply:
Yes, this section can be deleted. No need to point to MDN. Edge is supported on all platforms now. It used to be necessary to get a Windows VM through a service like BrowserStack if you didn't have a Windows device yourself in order to test EdgeHTML. That's no longer relevant though.